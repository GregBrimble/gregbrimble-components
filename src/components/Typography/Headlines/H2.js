import React from 'react';

import styled from 'styled-components';
import {Headline} from './shared';

const H2 = props => <Headline as={styled.h2`
    font-size: 3.75rem;
    line-height: 3.75rem;
`} {...props} />;

export default H2;
