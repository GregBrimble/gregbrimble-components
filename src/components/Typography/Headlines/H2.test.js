import React from "react"
import H2 from "./H2"
import renderer from "react-test-renderer"

describe("H1", () => {
  it("renders properly", () => {
    const tree = renderer.create(<H2>Headline 2</H2>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
