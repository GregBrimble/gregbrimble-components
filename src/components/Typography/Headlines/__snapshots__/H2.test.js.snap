// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 renders properly 1`] = `
<h2
  className="sc-bdVaJa hEOvXI sc-htpNat hedpSf"
>
  Headline 2
</h2>
`;
